src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+:root {
+  --header-height: 60px;
+  --underline-height: 4px;
+}
+
+body {
+  margin: 0 50px;
```

Comment:
Your header is tilted to right, most likely because you didn't reset margin of the body to 0, and set it to some value.

Reply:
I added correct padding to header.